meshpy/four_c/solid_shell_thickness_direction.py

```diff
@@ -207,7 +207,10 @@ def set_solid_shell_thickness_direction(
     for element in elements:
         is_hex8 = isinstance(element, VolumeHEX8)
         if is_hex8:
-            is_solid_shell = "SOLIDSH8" in element.dat_pre_nodes
+            is_solid_shell = (
```

Comment:
That is the only one I don't understand, since we check there the element is of `VolumeHEX8` type which has the `element.dat_pre_nodes` variable. I added an exception here.